## @antoinezl

My name is Antoine Lu and I have a Master's degree in Bioinformatics from [Université Paris Diderot](https://formation.univ-paris-diderot.fr/formations/master-biologie-informatiquebioinformatique).  
For more information about myself, please check out the left pannel!

---

## End of studies project:
### 6 months internship at [INRA-BIOGER](https://www6.versailles-grignon.inra.fr/bioger) (Plaisir Grignon, AMPC Team) under the supervision of [Nicolas Lapalu](https://github.com/nlapalu) and [Richard O'Connell](https://www.researchgate.net/profile/Richard_OConnell).

The aim of the internship was to test different genome assembly tools (PacBio's HGAP4 pipeline using Falcon and  which then allowed to set an assembly pipeline for two fungi (*Colletotrichum*).\
At the term of the internship, the genome of one species was fully assembled whereas the second one is almost assembled.  
Transcriptomics analysis () were executed (using RNA-seq data) for the fully assembled genome.  
A genome browser as been set up using the 'myGenomeBrowser' platform (only available for the APMC Team members at the moment).  
Finally, a internal website has been developped in HTML5/CSS/jQuery to present the data and results obtained during the internship.

The developped tools and the website are located on the 'develop' branch of my supervisor's Github, [**HERE**](https://github.com/nlapalu/BioinfobiogerWebGenome/tree/develop)

---

## "Projet Long"
### RNA Feature extraction, programming project in collaboration with PTER and [Costas Bouyioukos](https://github.com/cbouyio).

As part of the **Master's courses**, I collabored with the **PTER (Post Transcription ) Team** at Université Paris Diderot. The aim of the project was **to develop a tool that can allow one to extract defined features having a key role in determining whether a mRNA is commonly regulated or not**. This collaboration took place during the first semester of my second year of Master's degree. 

I was able to deliver a RNA feature extraction tool which takes **ENSEMBL ids** as input (as a list of ids) or a unique ID. The tool consists in 2 seperate **Python scripts** (**geneID2fasta.py** and **fasta2table.py**) and a library of functions called by the two previous scripts (**rnaFeaturesLib.py**).

The tool returns large datasets of RNA features (Kozak sequence and context, folding energy of the UTRs, length of each UTRs, etc.) that can be used for further analyses.

**_NOTE_**: The code (regulary updated, as other students are now working on the project) is available [**HERE**](https://github.com/parisepigenetics/rna_feat_ext/tree/develop_consolidation/mouse). 

---
